### PR TITLE
Small changes to manual testing

### PR DESCRIPTION
**Pull request by Pughal77**
We shoud remove usages of AddressBook and update it to references to our application, Avengers Assemble

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1540,24 +1540,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **MSS:**
 
 1.  User requests to edit a specific person with updated details.
-2.  AddressBook saves the updated details.
-3.  AddressBook confirms the successful update.
+2.  AvengersAssemble saves the updated details.
```
Comment (Zer0Legion) [inline docs/DeveloperGuide.md:1543]: Good spot. We should remove references to address book.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -2605,31 +2608,31 @@ On Window systems, you can right click the file and copy the file path, remember
...
 
 3. Adding an Exam with Missing Fields
 
    * **Prerequisites:** 
-       * No exams in the address book.
+       * No exams in the exams list.
          <br><br>
    * **Test case (missing score):** `addExam n|Final`<br>
-     **Expected:** Error message shown in the error report. No change in the address book.
+     **Expected:** Error message shown in the error report. No change in the exams lsit.
```
Comment (danielleloh) [inline docs/DeveloperGuide.md:2635]: lsit -> list

**Review by danielleloh — COMMENTED** [review]
typo errors

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1688,33 +1688,33 @@ into user's clipboard.
...
 
-    *  2a1. AddressBook displays a message indicating that the file is not recognised.
+    *  2a1. AvengersAssemble displays a message indicating that the file is not recognised.
 
        Use case ends.
 
 *   2b. The file to be imported is not a csv file.
 
-    *  2b1. AddressBook displays an error message indicating that the file type is not recognised and should be a csv file
+    *  2b1. AvengersAssemble displays an error message indicating that the file type is not recognised and should be a csv file
```
Comment (danielleloh) [inline docs/DeveloperGuide.md:1705]: csv -> CSV

**Review by danielleloh — COMMENTED** [review]
i think you should capitalise CSV

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -2093,40 +2096,40 @@ On Window systems, you can right click the file and copy the file path, remember
...
 
     <br>
 
-   * **Test case:** `import i|C:\file.csv`<br>
+   * **Test case:** `import i|C:\path\to\file.csv`<br>
      **Expected:** Error message shown in the error report. No change in list of persons.
 
    <br>
 
 3. Importing Data from a CSV File that is not a CSV File
```
Comment (danielleloh) [inline docs/DeveloperGuide.md:2108]: what does this mean ㅠㅠ

**Review by danielleloh — COMMENTED** [review]
minor catches

**Review by jayllo-c — COMMENTED** [review]
LGTM

**Comment by danielleloh** [comment]
LGTM too

**Pull request merged**